login-workflow/example2/src/navigation/Routing.tsx

```diff
@@ -47,6 +53,20 @@ export const GetCustomRoutes = (isAuthenticated: boolean) => {
                 </ExperimentalGuestGuard>
             ),
         },
+        // Authenticated Route: accessible only if the user IS authenticated
```

Comment:
not a big deal since its the example project, but this isn't an authenticated route